general/administration/installing.md

```diff
@@ -501,6 +507,41 @@ Finally you can run it. You will see lots of log information when run, this is n
 ./jellyfin.sh
 ```
 
+##### Starting Jellyfin on boot (optional)
+
+Create a `systemd` unit file.
+
+```sh
+cd /etc/systemd/system
+sudo nano jellyfin.service
+```
+
+Then paste the following contents, replacing `youruser` with your username.
+
+```
+[Unit]
+Description=Jellyfin
+After=network.target
+
+[Service]
+Type=simple
+User=youruser
+Restart=always
+ExecStart=/opt/jellyfin/jellyfin.sh
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Apply the correct permissions to the file, enable the service to start on boot, then start it.
+
+```sh
+sudo chmod 644 jellyfin.service
+sudo systemctl daemon-reload
+sudo systemctl enable jellyfin.service
+sudo systemctl start jellyfin.service
```

Comment:
```suggestion
sudo systemctl enable --now jellyfin.service
```

You can do that nowadays, but that's nitpicky lol

Reply:
I considered doing that, but thought it would be less explicit than writing out both commands. Anything works :)

Reply:
No your reason makes sense :)